src/app/components/Account/index.tsx

```diff
@@ -150,7 +150,7 @@ export const Account: FC<AccountProps> = ({ account, isLoading, tokenPriceInfo,
           <dt>{t('common.transactions')}</dt>
           <dd>
             {account.stats.num_txns ? (
-              <Link component={RouterLink} to={transactionsAnchor!}>
+              <Link component={RouterLink} to={transactionsAnchor!} replace>
```

Comment:
Bug on search page:
- open https://c4126abe.oasis-explorer.pages.dev/search?q=0xC3ecf872F643C6238Aa20673798eed6F7dA199e9
- click Transactions: 5
- go back

Reply:
Does `showLayer` prop meant to be used for such comparisons?

Reply:
That seems unreliable. I think a maintainable solution would be to automatically `replace` everytime location.pathname+location.search stays the same and only location.hash changes

Reply:
1) Why controlling this via props passed from `SearchResultsList` is not reliable? 
2) `location.pathname+location.search stays the same and only location.hash changes`. 
but location.pathname can change. user can nav through tabs and click transactions link

Reply:
Please note that since merging #634, tab navigation [no longer](https://github.com/oasisprotocol/explorer/pull/634/commits/4befa9eaaa7ffca50dda63072a0c975b386ca494) participates [in auto-scrolling](https://github.com/oasisprotocol/explorer/pull/634/commits/4322a1b7969cbc6f6e726eac6e40324d1687af4d).

Reply:
- Not reliable: showLayer has the wrong responsibility, so another prop would be better. But I think it isn't reliable because reusing the component in a different path can introduce a hardly detectable bug. I would rather have a path convention to define this behaviour, instead of each component call site.
- Path: oh, we don't even want to save tabs? I don't see an easily maintainable solution; can we just not do it?